app/utils/monitoring.server.ts

```diff
@@ -0,0 +1,9 @@
+import * as Sentry from '@sentry/remix'
+
+// TODO: Add Prisma integration
```

Comment:
For some reason, the Prisma integration was causing deployments to fail. I didn't spend a lot of time on this but we probably can leave it out for now

Reply:
How would this change if I wanted to try the prisma integration myself?